employee.rb

```diff
@@ -0,0 +1,19 @@
+class Employee
+  attr_accessor :name, :department, :review, :good_employee
+  attr_reader  :salary
+
+  def initialize(name, salary)
+    @salary = salary
+    @name = name
+    @department = nil
```

Comment:
Setting these two to nil isn't necessary, since all instance variables are nil if you haven't defined them yet.